### PR TITLE
feat: add yaml dependency

### LINKED ISSUE #272: Add yaml package
## why
Many scripts now depend on parsing and writing yaml files and/or artifacts, having native support for it would be great :]

## how
Add inline support to the [yaml](https://www.npmjs.com/package/yaml) package, as it was done with the [chalk](https://www.npmjs.com/package/chalk) one.

### PR DESCRIPTION
**Pull request by henrycunh**
Fixes #272 

- [x] Tests pass
- [x] Appropriate changes to README are included in PR

### CODE REVIEW

File: index.mjs [hunk 1]
```diff
@@ -32,6 +33,7 @@ export const globby = Object.assign(function globby(...args) {
   return globbyModule.globby(...args)
 }, globbyModule)
 export const glob = globby
+export const yaml = { parse: yamljs.parse, stringify: yamljs.stringify }
```
Comment (antonmedv) [inline index.mjs:36]: Let's export everything. And let's use `YAML` same as `JSON`.

File: index.mjs [hunk 2]
```diff
@@ -22,6 +22,7 @@ import {createInterface} from 'readline'
 import {default as nodeFetch} from 'node-fetch'
 import which from 'which'
 import chalk from 'chalk'
+import yamljs from 'yaml'
```
Comment (antonmedv) [inline index.mjs:25]: Don't forget to upgrade .d.ts as well.

**Review by antonmedv — CHANGES_REQUESTED** [review]

File: index.mjs (hunk 1 above)
Comment (henrycunh) [reply index.mjs]: ✅

File: index.mjs (hunk 2 above)
Comment (henrycunh) [reply index.mjs]: ✅

**Comment by henrycunh** [comment]
@antonmedv can we merge this little guy?

**Review by antonmedv — APPROVED** [review]

**Comment by antongolub** [comment]
@henrycunh, @antonmedv,

Why `yaml` and not `js-yaml`?
name | deps | stars | dloads 
--- | --- | --- | ---
yaml | 2762 | 521 | 18M
yaml-js | 14100 | 5,4K | 43M

**Comment by antonmedv** [comment]
I didn’t check it. Let’s switch!

**Comment by henrycunh** [comment]
@antongolub I thought about maintaining a API more similar to JS's JSON API (`parse` and `stringify`). 

`js-yaml` looks a lot like the Python's API, and imho, has worse documentation.

**Comment by antonmedv** [comment]
I thought they was the same. I also like JSON .parse api.

**Pull request merged**